src/plugins/recordTypes/transport/optionLists.js

```diff
@@ -0,0 +1,65 @@
+import { defineMessages } from 'react-intl';
+
+export default {
+  transportMethodType: {
+    values: [
+      'Cargo aircraft',
+      'Combi aircraft',
+      'Common carrier',
+      'Exclusive-use truck',
+      'Expedited use freight',
+      'LOFO freight',
+      'Mail',
+      'Non-commercial carrier',
+      'Ocean freight',
+      'Passenger aircraft',
+      'Shuttle service',
+    ],
```

Comment:
For static option lists, the best practice is to make the data values identical to the default message string, including case. That way you get exactly the same value if you're querying the database or REST API as you do in the UI, as long as you haven't overridden the default message. So it's best to lowercase all the above values (except for "LOFO freight" -- see comment below).